src/web/mjs/connectors/Delitoon.mjs

```diff
@@ -0,0 +1,66 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Delitoon extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'delitoon';
+        super.label = 'Delitoon';
+        this.tags = [ 'webtoon', 'french' ];
+        this.url = 'https://www.delitoon.com';
+        this.links = {
+            login: 'https://www.delitoon.com/connexion'
+        };
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.serie-page div.container div.informations-part1.detail div div.bloc-left h2');
+        return new Manga(this, uri.pathname, data[0].textContent.trim());
+    }
+    
+    async _getMangas() {
+        let mangaList = [];
+        let genres = ['Romance','BL','Drama','Sentimental','Historique','Slice%20of%20Life','Fantastique','Comédie','Thriller','Action','Aventure','SF'];
+        for(let page = 0; page < genres.length; page++) {
+            let mangas = await this._getMangasFromPage(genres[page]);
+            mangaList.push(...mangas);
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        let request = new Request(this.url + '/genres/' + page, this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.genres-page div.content ul li');
```

Comment:
May use `div.genres-page div.content ul li div.container-infos h3` to get all manga titles and then on each element  `element.closest('a')` to get the link.

Reply:
Suggestion did not work, link and title are not nested, they are in separate branches in the DOM